src/Command/Redis/QUIT.php

```diff
@@ -9,14 +9,16 @@
  * file that was distributed with this source code.
  */
 
-namespace Predis\Command;
+namespace Predis\Command\Redis;
+
+use Predis\Command\Command as RedisCommand;
 
 /**
  * @link http://redis.io/commands/quit
  *
  * @author Daniele Alessandri <suppakilla@gmail.com>
  */
-class ConnectionQuit extends Command
+class QUIT extends RedisCommand
```

Comment:
Why these class names?
I would recommend sticking to the naming defaults here (Quit, ...)

Reply:
Mostly because of case-sensitivity issues with autoloading.

With the [current implementation in `Predis\Command\RedisFactory`](https://github.com/predis/predis/blob/4b2b35202acbaf79c2f543fc0be1d3520acdbad3/src/Command/RedisFactory.php#L36-L49) the command ID is uppercased in order to follow the (mostly practical) convention used by Redis and have a direct lookup in the internal `$commands` dictionary, then the same uppercased string is used as-is by `class_exists()`. While it's ture that class names are treated as case insensitive by the PHP engine, PSR-4 enforces case sensitivity for autoloading so it'd break.

That said, and considered I'm not exactly satisfied with having a 1-1 match between Redis command ID and class name due to current (and potentially more, in the future) clashes with PHP keywords, it would probably make sense to have class names named like `GetCommand` instead of `GET` (albeit the Command part seems a bit redundant). Class names are incrementally cached by the internal `$commands` dictionary so doing a bit more stuff with the string to infer the actual class name would not really have an impact with performance (I always think about cases such as hefty pipelines where thousands of instances of commands can be created).

Reply:
That was my first thought as well `GetCommand` over `GET`, but we can change that after merging.